crates/eips/src/eip4844.rs

```diff
@@ -32,6 +34,21 @@ pub const BLOB_TX_MIN_BLOB_GASPRICE: u128 = 1u128;
 /// Commitment version of a KZG commitment
 pub const VERSIONED_HASH_VERSION_KZG: u8 = 0x01;
 
+/// How many bytes are in a blob
+pub const BYTES_PER_BLOB: usize = 131072;
```

Comment:
this is `DATA_GAS_PER_BLOB`

Reply:
I know—added an extra comment. I'll keep it as `DATA_GAS_PER_BLOB` is an u64, and i'd rather avoid casting everywhere we need this to be an usize.